src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
+/* eslint-disable */
 
 class Animal {
-  // write your code here
+  static newBeast(newAnimal) {
+    this.alive.push(newAnimal)
+  }
```

Comment:
this method is redundant, you can simply use push after setting needed properties in constructor